Src/FluentAssertions/Formatting/Formatter.cs

```diff
@@ -169,7 +170,7 @@ private static void Format(object value, FormattedObjectGraph output, Formatting
     /// </remarks>
     public static void RemoveFormatter(IValueFormatter formatter)
     {
-        CustomFormatters.Remove(formatter);
+        FormattingOptions.CustomFormatters.Remove(formatter);
```

Comment:
Maybe this should look like:

```csharp
FormattingOptions.RemoveCustomFormatter(formatter);
// and respectively
FormattingOptions.AddCustomFormatter(formatter);
```

For this I would then add this two extra static internal methods